performance/implementations/glamor/Box/index.js

```diff
@@ -6,12 +6,7 @@ import View from '../View';
 const Box = ({ color, fixed = false, layout = 'column', outer = false, ...other }) => (
   <View
     {...other}
-    style={[
-      styles[`color${color}`],
-      fixed && styles.fixed,
-      layout === 'row' && styles.row,
-      outer && styles.outer
-    ]}
+    style={[styles[`color${color}`], fixed && styles.fixed, layout === 'row' && styles.row, outer && styles.outer]}
```

Comment:
We also ignore imports and require > 80 lines at fb. There's an issue opened for that on prettier in order not to break them into multiple lines.

As for using it at Twitter, some thoughts:
- it is still very early and hasn't been battle tested in prod with fb employees. I'm only adding it to the codebase.
- if you could run the command with --debug-check instead of --write on your codebase, it does prettyprint(prettyprint(file)) === prettyprint(file). I would love if you could see if there's any that trigger on twitter codebase so that we can fix them.
- the printer itself at this point is pretty solid, there's still edge cases reported frequently but they are now pretty small and usually easy to fix.
- the biggest shaky piece is integration. Do you integrate it as a lint rule, as a precommit hook...? How do you keep the editor version with the linter version in sync? many of the integrations are currently calling the node "binary" every time which is slower as you have to pay for node startup time + parsing all the files...
- I'm working on prettier full time until it is widely used at Facebook, so expect a lot of polish in the next month or two :)